docs/source/deployment/databricks.md

```diff
@@ -223,3 +223,71 @@ Out[12]: {}
 Your complete notebook should look similar to this (the results are hidden):
 
 ![](../meta/images/databricks_notebook_example.png)
+
+
+### 9. Using IPython Extension 
+
+You can interact with Kedro in Databricks through the Kedro [IPython extension](https://ipython.readthedocs.io/en/stable/config/extensions/index.html), `kedro.extras.extensions.ipython`.
+
+The Kedro IPython extension launches a [Kedro session](../kedro_project_setup/session.md) and makes available the useful Kedro variables `catalog`, `context`, `pipelines` and `session`. It also provides the `%reload_kedro` [line magic](https://ipython.readthedocs.io/en/stable/interactive/magics.html) that reloads these variables (for example, if you need to update `catalog` following changes to your Data Catalog).
+
+The IPython extension can be used in a Databricks notebook in a similar way to how it is used in [Jupyter notebooks](https://kedro.readthedocs.io/en/0.18.0/tools_integration/ipython.html).
+
+Currently in Databricks, when you try to do a kedro run, it throws you a `ContextualVersionConflictError` because it downgrades `pip` to 21.0.1. We are not sure why this happens and hence there's one additional step you need to do in the Databricks notebook to make use of the IPython extension. After you load the IPython extension using the below command:
+
+```ipython
+In [1]: %load_ext kedro.extras.extensions.ipython
+```
+
+You must explicitly upgrade your `pip` version by doing the below so it is on the latest version `pip ~= 22.0.4`. 
+
+```bash
+%pip install -U pip
+```
+
+After this, you can reload Kedro by running the line magic command `%reload_kedro <path_to_project_root>`.
+
+### 10. Running Kedro-Viz on Databricks 
+
+Currently, if you try to run `%run_viz` on Databricks it will only display the below instead of running the Kedro-Viz app in the notebook. 
```

Comment:
`%run_viz` isn't a command available. I got this error, do I need to load it manually?

`UsageError: Line magic function `%run_viz` not found.`

Reply:
Hi Nok, this is what i thought would happened. It's because kedro, and kedro-viz are installed in your databricks seperately. You will either have to uninstall kedro from the Cluster Library like i showed you and then do pip i kedro. Or install kedro-viz on the cluster library. if you do tht, run_viz should work. I will also highlight this in the docs!!!

Reply:
Definitely worth noting this I think because it's not at all obvious. Something like this:
> Make sure Kedro-Viz and Kedro are both installed in the same scope (notebook-scoped vs. cluster library). i.e. if you `%pip install kedro` from inside your notebook then you should also `%pip install kedro-viz` from inside your notebook. If your cluster comes with Kedro installed on it as a library already then you should also [add Kedro-Viz as a cluster library](https://docs.microsoft.com/en-us/azure/databricks/libraries/cluster-libraries).

Reply:
yes, will add this